src/main/java/com/vu/aezakmi/multipartRequest/VideoUploadRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.vu.aezakmi.multipartRequest;
+
+import com.vu.aezakmi.dto.VideoDto;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.web.multipart.MultipartFile;
+
+@Getter @Setter
+public class VideoUploadRequest {
+    private MultipartFile file;
+    private VideoDto videoDto;
```

Comment:
hmm, client would have to pass `videoDto.title`, `videoDto.description` in its POST body. Would be nicer to have just straight up title, description, etc. in the body.

Reply:
also, video DTO still has fileURL as one of its fields